dev-python/PyQt5/metadata.xml

```diff
@@ -25,7 +25,6 @@
 		<flag name="svg">Build bindings for the QtSvg module</flag>
 		<flag name="testlib">Build bindings for the QtTest module</flag>
 		<flag name="webchannel">Build bindings for the QtWebChannel module</flag>
-		<flag name="webkit">Build bindings for the QtWebKit module</flag>
```

Comment:
…but the flag is still used in the old version, isn't it?

Reply:
Yes it is, good point.